src/OrbitBase/include/OrbitBase/SpanUtils.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_SPAN_UTILS_H_
+#define ORBIT_BASE_SPAN_UTILS_H_
+
+#include <absl/types/span.h>
+
+namespace orbit_base {
+
+// Create spans of specified size from an input std::vector. Note that the last span can be smaller
+// than the specified size if the input vector size is not a multiple of the span size.
+//
+// Example: Splitting data in spans that can be processed in parallel tasks:
+//
+// void ProcessObjectsInParallel(std::vector<Object>& objects) {
+//   TaskGroup task_group;
+//   for (absl::Span<Object> span : CreateSpansOfSize(objects, 1024)) {
+//     task_group.AddTask([span]() {
+//       for (Object& object : span) {
+//         Process(object);
+//       }
+//     });
+//   }
+// }
+//
+template <typename T>
+[[nodiscard]] inline std::vector<absl::Span<T>> CreateSpansOfSize(std::vector<T>& input_vector,
```

Comment:
Nit: `inline` is implicit for templates.

Reply:
Ah yes, of course, thanks.